tq/verify.go

```diff
@@ -33,10 +40,18 @@ func verifyUpload(c *lfsapi.Client, t *Transfer) error {
 	}
 	req.Header.Set("Content-Type", "application/vnd.git-lfs+json")
 
-	res, err := c.Do(req)
-	if err != nil {
-		return err
-	}
+	mv := c.GitEnv().Int(maxVerifiesConfigKey, defaultMaxVerifyAttempts)
+	mv = tools.MaxInt(defaultMaxVerifyAttempts, mv)
 
-	return res.Body.Close()
+	for i := 1; i <= mv; i++ {
+		tracerx.Printf("tq: verify %s attempt #%d (max: %d)", t.Oid[:7], i, mv)
+
+		var res *http.Response
+
+		if res, err = c.Do(req); err == nil {
```

Comment:
I think it'd be a good idea to print the message to tracerx, in case someone's having issues and we want to see the errors.